client/dataEntry.js

```diff
@@ -31,20 +31,12 @@ Template.dataEntry.helpers({
   },
 
   isSelectedRoundClosed: function() {
-    if(!this.roundId) {
-      // If there's no round selected, then the selected round is definitely
-      // *not* closed =)
-      return false;
-    }
-    return Rounds.findOne(this.roundId).isClosed();
+    round = Rounds.findOne(this.roundId);
```

Comment:
Hmm... this refactor does change the behavior of the code a little. Previously we would barf if `this.roundId` corresponded to a nonexistant Round. Now we silently ignore it. How about:

```javascript
return this.roundId ? Rounds.findOne(this.roundId).isClosed() ? false;`
```